lib/studies/crane/backdrop.dart

```diff
@@ -71,11 +74,7 @@ class _FrontLayerState extends State<_FrontLayer> {
   Widget build(BuildContext context) {
     final isDesktop = isDisplayDesktop(context);
     final isSmallDesktop = isDisplaySmallDesktop(context);
-    final crossAxisCount = isSmallDesktop
-        ? 2
-        : isDesktop
-            ? 4
-            : 1;
+    final crossAxisCount = isDesktop ? 4 : 1;
```

Comment:
Why the change?

Reply:
Didn't look right with 2 columns, way too much whitespace, now everything always has the same spacing